CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,130 @@
+i# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
```

Comment:
Should we add, that this code of conduct holds for all repositories within the SeisSol project?

Reply:
No, I'd just push it to other repositories later - otherwise it becomes very confusing. Maybe add it to the website as well (at some point).

Reply:
At least if we interpret community as the seissol community - which is fair here, I think, because this is obviously the main repository. Or what do you think?

Reply:
So, let's have it at the main repo + website and link to it from all the other repos?

Reply:
Yup, that's a good idea. Not sure whether we need to link it from the other repos. I guess, most people come to the main repo first.

Reply:
Hi. @krenzland thanks for your work. Maybe we can do something what [spack](https://github.com/spack/spack#code-of-conduct) did. I mean we can make a link to code of conduct in our main README.md. Would it be better if the file was placed in `<root>/.github` directory?

Reply:
Yes, it's a good idea to add this to the README.md.

About the .github directory: Github detects the file also in this way and it is a bit more visible when it is in the root.